content/docs/command-reference/exp/run.md

```diff
@@ -0,0 +1,108 @@
+# exp run
+
+Run an experiment: reproduce a variation of a committed
+[pipeline](/doc/command-reference/dag) in a hidden project branch.
+
+> Similar to `dvc repro` but for
+> [experimentation](/doc/user-guide/experiment-management).
+
+## Synopsis
+
+```usage
+usage: dvc exp run [-h] [-q | -v] [-f]
+                   [<repro_options> ...]
+                   [--params [<filename>:]<params_list>]
+                   [-n <name>] [--queue] [--run-all] [-j <number>]
+                   [targets [targets ...]]
+
+positional arguments:
+  targets               Stages to reproduce. 'dvc.yaml' by default.
+```
+
+## Description
+
+Provides a way to execute and track `dvc experiments` in your
+<abbr>project</abbr> without polluting it with unnecessary commits, branches,
+directories, etc.
+
+> `dvc exp run` is equivalent to `dvc repro` for <abbr>experiments</abbr>. It
+> has the same behavior when it comes to stage execution (restores the
+> dependency graph, etc.). It differs from in that experiment dependencies and
+> outputs will always be <abbr>cached</abbr> (so they can be restored later).
+
+Each `dvc exp run` creates a variation based on the last project version (Git
+commit) and stores it internally with an automatic experiment name like
```

Comment:
Suggestion to drop "project version" as it might just add confusion.

```suggestion
Each `dvc exp run` creates a variation based on the last Git
commit and stores it internally with an automatic experiment name like
```

Reply:
How about "latest project version committed to Git"? I know it's long but the project version concept is more important I think, (motivation), while Git commit part is a technical requirement also worth mentioning.

Reply:
Sure, I have no strong feelings on this, so feel free to leave as is or make whatever change you like.

Reply:
Already updated 👍